sw_stm32/Communication/uSD_handler.cpp

```diff
@@ -376,6 +386,7 @@ void write_magnetic_calibration_file ( void)
         return;
     }
 
+#if USE_EARTH_INDUCTION_DATA_COLLECTOR
```

Comment:
What is the use case of disabling the nav induction report`

Reply:
It seems that this is disabled by default.  I prefer having less precompiler flags to keep readability complexity low.